sky/cli.py

```diff
@@ -2316,7 +2331,7 @@ def _down_or_stop_clusters(
                 # TODO(zhwu): We can only have one reserved cluster (spot
                 # controller).
                 assert len(reserved_clusters) == 1, reserved_clusters
-                _hint_for_down_spot_controller(reserved_clusters[0])
+                _hint_or_raise_for_down_spot_controller(reserved_clusters[0])
 
                 if not no_confirm:
                     click.confirm('Proceed?',
```

Comment:
One option, for discussion: we can check
```
value = click.prompt('Type 'delete' to proceed', type=str)
if value == 'delete':
  ...
else:
  raise click.Abort() from None
```

Reply:
Great point! I changed it to the following. PTAL. : )
<img width="680" alt="image" src="https://user-images.githubusercontent.com/6753189/217997444-eeb2a252-f266-4364-8b5f-970027bdfcda.png">

Reply:
Kind of feel this is a bit too long; was proposing `delete` since it's used in AWS console for certain VPC resources. Wdyt? Feel free to do a poll too.

Reply:
Good point! I used the `I understand` because that will remind the user to read the messages above, but I am not feeling strongly about this.  Switched back to `delete`.